### PR TITLE
Added base conversions.

### PR DESCRIPTION
**Pull request by shivhek25**

### CODE REVIEW

**Comment by dynamitechetan** [comment]
@shivhek25  please check the merge conflicts

File: Bubble Sort.java [hunk 1]
```diff
@@ -2,22 +2,24 @@
 
 class BubbleSort
 {
-	public static void main(String[] args) 
+	public static void main(String[] args)
 	{
-		int array[]=new int[6];
+		int n;
+		int array[]=new int[n];
```
Comment (DocJava) [inline Bubble Sort.java]: "n" has no value here, this will break

File: Bubble Sort.java [hunk 2]
```diff
@@ -2,22 +2,24 @@
...
 		Scanner input=new Scanner(System.in);
-
+    System.out.println("Enter length of array:");
+		n=sc.nextInt();
```
Comment (DocJava) [inline Bubble Sort.java]: now that n has a value we can initialize the "array" variable down here

File: Decimal to Hexadecimal.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+{
+    public static void main(String args[])
+    {
+        Scanner sc=new Scanner(System.in);
+        int n,k,i,d;
+        String s=" ";
+        int a[]={'A','B','C','D','E','F'};
+        n=sc.nextInt();
+        k=n;
+        while(k!=0)
```
Comment (DocJava) [inline Decimal to Hexadecimal.java]: infinite while loop, the value of k is never modified

**Comment by shivhek25** [comment]
Made the requisite changes. Kindly check.

File: Decimal to Hexadecimal.java (hunk 3 above)
Comment (shivhek25) [inline Decimal to Hexadecimal.java]: Updated it.

File: Bubble Sort.java (hunk 1 above)
Comment (shivhek25) [inline Bubble Sort.java]: Made the changes.

**Comment by shivhek25** [comment]
Can the admins and reviewers please look into the updations ? Thank you. @dynamitechetan @DocJava @AnupKumarPanwar

**Comment by DocJava** [comment]
Looks fine, I would suggest using readable variable names (instead of i k n j or other random letters).
I still vote taking a look at the Java code style guide https://google.github.io/styleguide/javaguide.html 
The GCD file (looks fine, I did not run it, but here are some notes): 
-The main method throws exception , but the entire code is caught in a try catch..so nothing would ever be thrown.
-I'm pretty sure the try catch would never actually catch anything? 
-The catch has "return" in it, but it is the last thing to happen in a block, so really you could just leave it empty or something and have the same effect?

**Comment by shivhek25** [comment]
Sure. Should I change the variable names for these codes as well, or should I keep this pointer in mind next time onwards ? @DocJava

**Comment by DocJava** [comment]
That is up to you, the coder. (I vote to change it now though)

**Comment by r0hit-gupta** [comment]
@shivhek25 Please resolve the conflicts.

**Comment by shivhek25** [comment]
I have already made all the changes. Could you please specify which conflicts are to be resolved ?

**Comment by 0oo0** [comment]
There are conflicts in BubbleSort.java
You can see them on Github.
Merge conflicts arise when you compete for the same line change by different branches. 
Try this
https://help.github.com/articles/resolving-a-merge-conflict-on-github/

**Comment by shivhek25** [comment]
The resolve conflicts is deactivated so I followed the instructions on how to resolve the error using cmd. After having done those, the file is still under conflict.

**Comment by dynamitechetan** [comment]
if you are not able to resolve the conflicts then close this PR and send a new one with your changes.

**Pull request closed without merging**